lib/moonshot/stack.rb

```diff
@@ -291,17 +295,22 @@ def wait_for_stack_state(wait_target, past_tense_verb)
       result
     end
 
-    def make_tags
+    def self.make_tags(config)
       default_tags = [
-        { key: 'moonshot_application', value: @config.app_name },
-        { key: 'moonshot_environment', value: @config.environment_name }
+        { key: 'moonshot_application', value: config.app_name },
+        { key: 'moonshot_environment', value: config.environment_name },
       ]
+      name=Stack.generate_name(config)
```

Comment:
You can skip the Stack cal as you are calling a class method from a class method. nit: space around '=''.
```suggestion
      name = generate_name(config)
```